aucoalesce/normalizations.yaml

```diff
@@ -45,708 +71,1076 @@ macros:
-    primary: new-level
-    what: system
-  record_types: SYSTEM_RUNLEVEL
--
-  action: shutdown-system
-  object:
-    what: system
-  record_types: SYSTEM_SHUTDOWN
--
-  action: sent-test
-  record_types: TEST
--
-  action: unknown
-  record_types: TRUSTED_APP
--
-  action: sent-message
-  object:
-    primary: addr
-  record_types: USER
--
-  <<: *macro-user-session
-  action: was-authorized
-  record_types: USER_ACCT
--
-  <<: *macro-user-session
-  action: authenticated
-  record_types: USER_AUTH
--
-  action: access-permission
-  record_types: USER_AVC
--
-  <<: *macro-user-session
-  action: changed-password
-  record_types: USER_CHAUTHTOK
--
-  action: ran-command
-  object:
-    primary: cmd
-    what: process
-  record_types: USER_CMD
-  description: >
-    These messages are from user-space apps, like sudo, that log commands
-    being run by a user. The uid contained in these messages is user's UID at
-    the time the command was run. It is not the "target" UID used to run the
-    command, which is normally root.
--
-  <<: *macro-user-session
-  action: ended-session
-  record_types: USER_END
--
-  <<: *macro-user-session
-  action: error
-  record_types: USER_ERR
-  source_ip: [addr]
--
-  <<: *macro-user-session
-  action: logged-in
-  record_types: USER_LOGIN
-  source_ip: [addr]
--
-  <<: *macro-user-session
-  action: logged-out
-  record_types: USER_LOGOUT
--
-  action: changed-mac-configuration
-  record_types: USER_MAC_CONFIG_CHANGE
--
-  action: loaded-mac-policy
-  record_types: USER_MAC_POLICY_LOAD
--
-  <<: *macro-user-session
-  action: modified-user-account
-  record_types: USER_MGMT
--
-  <<: *macro-user-session
-  action: changed-role-to
-  object:
-    primary: selected-context
-    what: user-session
-  record_types: USER_ROLE_CHANGE
--
-  action: access-error
-  record_types: USER_SELINUX_ERR
--
-  <<: *macro-user-session
-  action: started-session
-  record_types: USER_START
-  source_ip: [addr]
--
-  action: changed-configuration
-  object:
-    primary: op
-    what: system
-  record_types: USYS_CONFIG
--
-  action: issued-vm-control
-  object:
-    primary: op
-    secondary: vm
-    what: virtual-machine
-  record_types: VIRT_CONTROL
--
-  action: created-vm-image
-  record_types: VIRT_CREATE
--
-  action: deleted-vm-image
-  record_types: VIRT_DESTROY
--
-  action: checked-integrity-of
-  record_types: VIRT_INTEGRITY_CHECK
--
-  action: assigned-vm-id
-  object:
-    primary: vm
-    what: virtual-machine
-  record_types: VIRT_MACHINE_ID
--
-  action: migrated-vm-from
-  record_types: VIRT_MIGRATE_IN
--
-  action: migrated-vm-to
-  record_types: VIRT_MIGRATE_OUT
--
-  action: assigned-vm-resource
-  object:
-    primary: resrc
-    secondary: vm
-    what: virtual-machine
-  record_types: VIRT_RESOURCE
-- action: typed
-  object:
-    primary: data
-    what: keystrokes
-  how: [comm, exe]
-  record_types:
-  - TTY
-  - USER_TTY
+  - ecs: *ecs-process
+    syscalls:
+      - '*' # this is a catch all
```

Comment:
The thought is that only a process can actually make syscalls anyway--so if we have a syscall where we haven't normalized it to a behavior (i.e. create a file)--then just say it's "process info" with ECS: `event.category: process, event.type: info`